src/main.cairo

```diff
@@ -180,6 +192,54 @@ func prolong_auction_on_end{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, ra
     return ()
 end
 
+func verify_auction_does_not_exist{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(auction_id):
```

Comment:
Rename all verify functions to assert and move to file assertions

Reply:
State will have to be moved to a different file for this reason, but this will simplify the code